### ISSUE TITLE
Build an l10n team repository

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#7717]
Build l10n team's fluent repository for Bedrock (waiting on some ftl files)

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Vendor and pontoon changes have a repository to write to
- [ ] Bedrock's automation is connected to that repository

**Comment by hoosteeno** [comment]
@pmac is there any progress on this one? Do we need anything to get it started?

**Comment by pmac** [comment]
This is pretty much all @Pike. He may be waiting until we get our first .ftl files merged into bedrock, but I think the ones in the [www-l10n repo](https://github.com/mozmeao/www-l10n) are good enough to get going.

**Comment by Pike** [comment]
Right now, there are conflicts between what I had in mind for the l10n repo and what PR #7679 does. I'm still working myself through that PR, but right now, the fluent files in bedrock aren't good to expose to l10n, which is what I could do. We'll need to work those out in the PR.

I'd also like to clarify that I won't be hooking up automation.

**Comment by pmac** [comment]
My PR works on our repo, your repo is another thing that our automation will manage transfers between. You don't have to take our ported files. We'll just be porting things as we need to. If there are problems with those files I think it's mostly due to poor source files.

**Comment by hoosteeno** [comment]
This issue depends on https://github.com/mozilla/bedrock/issues/7723

**Comment by Pike** [comment]
I created https://github.com/mozilla-l10n/www-l10n and set up a bunch of permissions. Also added COC and license file.

I think this should unblock things for now.

**Comment by hoosteeno** [comment]
This seems done -- is that true, @Pike ?

**Comment by alexgibson** [comment]
I'm going to close this issue as https://github.com/mozilla-l10n/www-l10n now exists and has an intake of content. Feel free to reopen if anything else needs doing here.
